app/routes/index.tsx

```diff
@@ -1,61 +1,12 @@
-import { LinksFunction, LoaderFunction } from '@remix-run/node';
-import styles from 'react-datepicker/dist/react-datepicker.css';
-import Calendar from '~/components/Calendar';
-import Navbar from '~/components/common/navbar';
-import axios from 'axios';
-import dayjs from 'dayjs';
-import { useStore } from '~/store/useStore';
-import { useLoaderData } from '@remix-run/react';
-import { parseEvents } from '~/utils/event.utils';
-import { getEvents } from '~/constants/urls.constants';
-import { ToastContainer, toast } from 'react-toastify';
-import { useEffect } from 'react';
+import { LoaderFunction, redirect } from '@remix-run/node';
 
-export let loader: LoaderFunction = async ({ request }) => {
-  const cookie = request.headers.get('cookie');
-
-  const startTime = dayjs().subtract(1, 'months').startOf('month').unix() * 1000;
-  const endTime = dayjs().add(1, 'months').endOf('month').unix() * 1000;
-
-  try {
-    const response = await axios.get(getEvents(process.env.API_HOST ?? '', startTime, endTime), {
-      headers: {
-        'Content-Type': 'application/json',
-        Cookie: cookie,
-      },
-    });
-    return { events: response.data.data, error: null };
-  } catch (error) {
-    return { events: null, error: error };
+export const loader: LoaderFunction = async ({ request }) => {
+  const url = new URL(request.url);
+  if (url.pathname === '/') {
+    return redirect('/calendar');
   }
+  return null;
 };
+const index = () => <></>;
```

Comment:
we need to look into ways to bring calendar on the index route. Using the root route to just redirect is a waste of the route and potentially problematic in terms of SEO.

@harshith-venkatesh

Reply:
yes, we can take that up in future
**index page for cron and vimcal**
1. https://cron.com/
2. https://www.vimcal.com/
I think we can do something like this in future